projects/fal/src/fal/cli/apps.py

```diff
@@ -255,6 +255,24 @@ def _runners(args):
 
     args.console.print(table)
 
+    requests_table = Table()
+    requests_table.add_column("Runner ID")
+    requests_table.add_column("Request ID")
+    requests_table.add_column("Caller ID")
+
+    for runner in runners:
+        for lease in runner.external_metadata.get("leases", []):
+            if not (req_id := lease.get("request_id")):
+                continue
+
+            requests_table.add_row(
+                runner.runner_id,
+                req_id,
+                lease.get("caller_user_id") or "",
+            )
+
+    args.console.print(requests_table)
```

Comment:
Hm, looks like we could just add it to the previous table, right? Not clear why we need a second one here.

Reply:
we may have severalr reqs per runner

Reply:
we can probably do better, but lets keep it this way for now 👍